Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG BASE_IMAGE_TAG="3.25.11-rc.870"
-FROM ghcr.io/xgovformbuilder/digital-form-builder-runner:$BASE_IMAGE_TAG as base
+ARG BASE_IMAGE_TAG="b833677ea06f6409f8555b2743f73bcbd05b6eba"
```

Comment:
Do we need something in the readme to explain how to update this?

Reply:
Done

Reply:
Done